src/include/storage/pmsignal.h

```diff
@@ -40,11 +40,14 @@ typedef enum
 	PMSIGNAL_BACKGROUND_WORKER_CHANGE,	/* background worker state change */
 	PMSIGNAL_START_WALRECEIVER, /* start a walreceiver */
 	PMSIGNAL_ADVANCE_STATE_MACHINE, /* advance postmaster's state machine */
+    	PMSIGNAL_FAILED_LOGIN,		/* send signal SIGUSR1 to postmaster */
 
 	PMSIGNAL_WAKEN_FTS,         /* wake up FTS to probe segments */
 	PMSIGNAL_WAKEN_DTX_RECOVERY,         /* wake up dtx recovery to abort dtx xacts */
 	PMSIGNAL_DTM_RECOVERED,     /* distributed recovery completed */
 
+	PMSIGNAL_START_LOGIN_MONITOR_WORKER,	/* start a login monitor worker */
+
```

Comment:
please add more description to `PMSIGNAL_FAILED_LOGIN` comments, and why not put them together to avoid ABI?

Reply:
I don't understand what the ABI you mean? Can you describe it？

Reply:
put `PMSIGNAL_FAILED_LOGIN` before `PMSIGNAL_WAKEN_FTS` will change the number value from `PMSIGNAL_WAKEN_FTS` to `PMSIGNAL_DTM_RECOVERED`, better to put  new fields append from the last -1 one if  there is no special reason.

Reply:
OK.